hm_pyhelper/miner_param.py

```diff
@@ -41,6 +45,11 @@ def run_gateway_mfr(args):
         err_str = "gateway_mfr exited with a non-zero status"
         LOGGER.exception(err_str)
         raise ECCMalfunctionException(err_str).with_traceback(e.__traceback__)
+    except (FileNotFoundError, NotADirectoryError) as e:
+        err_str = "file/directory for gateway_mfr was not found"
+        LOGGER.exception(err_str)
```

Comment:
Is there a pattern you are using for when to log errors or just raise them? We have been inconsistent about this up to now.

Reply:
I usually do both log and raise it, if the error is a known one rather than generic I log it with a specific message relating to the exception thrown and if it's unknown I just include the exception as a string